### PR TITLE
Add test repository create shop

### PR DESCRIPTION
**Pull request by stringtheory1234**

### CODE REVIEW

File: backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@SpringBootTest
+@RunWith(SpringRunner.class)
+@ContextConfiguration(classes = InMemoryVirtualQueueRepository.class)
+public class InMemoryVirtualQueueRepositoryTest {
+    InMemoryVirtualQueueRepository inMemoryVirtualQueueRepository = new InMemoryVirtualQueueRepository();
+    @BeforeEach
```
Comment (priytanvani) [inline backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java:36]: Nit. Add new line?

File: backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        inMemoryVirtualQueueRepository.getShopMap().clear();
+        inMemoryVirtualQueueRepository.getShopIdToListOfTokensMap().clear();
+        inMemoryVirtualQueueRepository.getShopIdToLastAllotedTokenMap().clear();
+        inMemoryVirtualQueueRepository.getShopOwnerMap().clear();
+    }
+
+    @Test
+    public void testCreateShop() throws Exception {
+        CreateShopRequest shop = new CreateShopRequest("uid", "shopName", "address", "+919012192800", "type");
+        UUID shopId = inMemoryVirtualQueueRepository.createShop(shop);
```
Comment (priytanvani) [inline backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java:48]: Nit. Add newline above this?

File: backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    }
+
+    @Test
+    public void testCreateShop() throws Exception {
+        CreateShopRequest shop = new CreateShopRequest(shopOwnerId, shopName, shopAddress, phoneNumber, shopType);
+
+        UUID shopId = inMemoryVirtualQueueRepository.createShop(shop);
+        int size = inMemoryVirtualQueueRepository.getShopMap().size();
+
+        assertEquals("size of shopMap", 1, size);
```
Comment (priytanvani) [inline backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java:59]: inline this variable as well like the others?

Comment (stringtheory1234) [reply backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java]: done!

File: backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@SpringBootTest
+@RunWith(SpringRunner.class)
+@ContextConfiguration(classes = InMemoryVirtualQueueRepository.class)
+public class InMemoryVirtualQueueRepositoryTest {
+    InMemoryVirtualQueueRepository inMemoryVirtualQueueRepository = new InMemoryVirtualQueueRepository();
+
+    private static final String shopOwnerId = "uid";
```
Comment (priytanvani) [inline backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java:37]: Nit. SHOP_OWNER_ID.
We generally try to name constants in capital with snake case.

File: backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+        inMemoryVirtualQueueRepository.getShopOwnerMap().clear();
+    }
+
+    @Test
+    public void testCreateShop() throws Exception {
+        CreateShopRequest shop = new CreateShopRequest(shopOwnerId, shopName, shopAddress, phoneNumber, shopType);
+
+        UUID shopId = inMemoryVirtualQueueRepository.createShop(shop);
+
+        assertEquals("size of shopMap", 1, inMemoryVirtualQueueRepository.getShopMap().size());
```
Comment (priytanvani) [inline backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java:58]: Nit. s/S ?

Comment (stringtheory1234) [reply backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java]: yeah changed it to S.

File: backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java (hunk 4 above)
Comment (stringtheory1234) [reply backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java]: changed all variables to snake case.

**Pull request merged**